README.md

```diff
@@ -14,9 +14,10 @@ billlanyon@me.com
 3. Make the game executable by entering 'chmod +x main.py'
 4. Launch the game by entering 'python3 main.py' and following the on screen instructions.
 5. Start a game by entering 'y'.
-6. Select a game against a computer opponent by entering 'c', or 'h' to play against another human.
-7. A human player always plays first: they chose their player by entering X or O, a space, and then a cell from 0 to 8.
-   The following players then just enter a coordinate for each move.
+6. Select a board size, from 3 - 9, by entering the dimension in letters, 'six' for example. The default board size is 3x3.
```

Comment:
why not just the number as an integer ?

Reply:
Because of the extant string based get_user_input function in main.

Reply:
Is this acceptable?

Reply:
I see what you mean.

How about doing the conversion in the get_board_size() method ?

`def get_board_size():
    return int(
                     get_user_input("What board dimensions will you play? Please enter a number from 3 to 9: ",
                          ['3', '4', '5', '6', '7', '8', '9'])
                    )
`

Perhaps put that in a try block in case of user silliness.

Reply:
Done and thank you for the suggestion: what a fool am I :).